bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,298 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=80,
+                        max_line_length_affects_comments: bool=True,
+                        space_before_and_after_arrow: bool=False,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=0,
+                        spacing_in_empty_braces: int=0,
+                        camelcase_classes_name: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        check_whether_file_ends_in_newline: bool=False,
+                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                        no_tabs: bool=True,
```

Comment:
we have a setting `use_spaces` in other bears and we want setting names to be consistent across bears ideally so the user can just set it once for multiple bears